posts/git-cherry-pick/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Cherry-Pick on GitHub"
+author: "Robyn Marowitz"
+date: "2023-09-26"
+description: "Committing a mistake in GitHub can feel so scary, cherry picking is a tool to help you fix your history when something like that happens. This post lays out simple and clear instructions on the cherry picking method."
+categories: [github, open-science]
+image: "cherry_pick.jpeg"
+---
+
+## What is git cherry-pick?
+
+It is the act of picking a commit from one branch and applying it to another branch.  
+
+## When should I use cherry-pick?
+
+Cherry-Pick is best used when a change has been committed onto a different branch than desired. You can *pick* commits one by one and move them to the desired branch. 
+
+`git cherry-pick` usually brings a commit from somewhere else and applies it on top of your current branch, recording a new commit, while `git rebase` takes your current branch and rewrites a series of its own tip commits in one way or another.
```

Comment:
Maybe a sub-header for this section? Could be `### How is cherry-pick different from rebase?`

Reply:
>`git rebase` takes your current branch and rewrites a series of its own tip commits in one way or another

`git rebase` can act on any branch actually! The docs and man pages have a concise description for `git rebase`: "Reapply commits on top of another base tip". Maybe we can paraphrase from that, but it'd be good to explain what's probably most important to the user: `cherry-pick` is _simpler_ and _less risky_. `rebase` is a valuable tool, but if you don't understand what it's doing, you can get in trouble pretty easily.

Might be worth mentioning that we'll cover it in a future post. Or try to :)